panorama_connector.py

```diff
@@ -1599,7 +1640,7 @@ def _run_query(self, param):
             time.sleep(2)
 
         try:
-            action_result.set_summary({'num_logs': int(result_data['log']['logs']['@count'])})
+            action_result.update_summary({'num_logs': int(result_data['log']['logs']['@count'])})
```

Comment:
We want to use `update_summary` here to include the results of API calls done in earlier steps.